docs/modules/ROOT/pages/how-tos/creating.adoc

```diff
@@ -1 +1,127 @@
-= Creating applications and components
\ No newline at end of file
+= Creating applications and components
+
+To create an application in {ProductName}, someone must enable build pipelines in your instance of {ProductName}. This includes creating a GitHub application, so {ProductName} can access users' pull requests, and configuring a smee channel, to listen for those PRs. The first procedure in this document explains how to perform these tasks. 
+
+Once build pipelines are enabled, users can create their applications in {ProductName}. The second procedure below describes how to create an application. 
+
+== Enabling build pipelines for your instance of {ProductName}:
+
+.Prerequisites:
+
+* Install link:https://github.com/konflux-ci/konflux-ci/tree/main[{ProductName}]
+* Read/write access to your {ProductName} instance’s repository
+
+.Procedure:
+
+. Create a GitHub app according to link:https://pipelinesascode.com/docs/install/github_apps/#manual-setup[these Pipelines as Code instructions].
+. Start a new link:https://smee.io/[smee] channel.
+. In the repository for your instance of {ProductName}, edit the file /smee/smee-client.yaml. Replace <smee-channel> with the webhook proxy URL from the previous step. 
+
+== Creating your applications and components with {ProductName}:
+
+.Prerequisites:
+
+* link:https://kubernetes.io/docs/tasks/tools/[kubectl] CLI tool
+* The following values from your {ProductName} GitHub application:
+** The private key
+** The application ID
+** The webhook secret
+* A GitHub repository with source code for your application 
+** It must have the same basic structure as our example repository
+* Read/write access to the repository for your instance of {ProductName} 
+
+.Procedure:
+
+. In the GitHub repository for your application, install your organization’s {ProductName} GitHub application.
+. In the repository for your instance of {ProductName}, you need to replace two `url` values in the relevant YAML file for your namespace. The path for this file is `/test/resources/demo-users/user/<namespace>/application-and-component.yaml`. Replace the two `url` values as follows:
+.. For the `url` value beneath `git`, provide the URL that you would use to locally clone your application’s repository. This URL ends with `.git`.
+.. For the `url` value beneath `spec`, at the end of the file, provide the URL for your application’s repository that you see in your browser, without `.git` at the end.
++
+NOTE: To add other components to the same namespace, reference the example YAML file beneath this procedure. 
+. Login to {ProductName} xref:../getting-started/cli.adoc[through your CLI]. 
```

Comment:
For this step, the user needs to be logged in to the kubernetes cluster where konflux is deployed rather than to konflux.